crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -304,6 +304,8 @@ pub enum Action {
     Transfer(TransferAction),
     ValidatorUpdate(tendermint::validator::Update),
     SudoAddressChange(SudoAddressChangeAction),
+    #[cfg(feature = "mint")]
```

Comment:
As above. This feature gate is going to infect every crate that somehow handles these actions.

On the one hand I like that this is unrepresentable, but on the other I think the handling of these mint actions should be entirely confined to the sequencer handlers

Reply:
see above - should i just ignore enum cases that are not enabled in the sequencer then?

Reply:
Yes, I think so. I would actually leave two mint components in: the deactivated one that just logs if we received a mint request (and maybe from where), and the feature gated one that has the actual functionality.